<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Тестовая программа анализа файлов прогноза</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь к каталогу с файлами прогноза</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnSelect">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnAnalize">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>105</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Анализ</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>481</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Найденные несоответствия:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>75</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>от</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEditFrom">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>70</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEditTo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>75</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>до</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>75</y>
      <width>47</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>шаг, ч</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxStep">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>47</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Прогноз:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCount">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>47</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Заблаговременность:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>от, ч</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>до, ч</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>47</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>шаг, мин</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxEFrom">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>105</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxETo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>105</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxEStep">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>105</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushBtnSelect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBtnAnalize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startAnalize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectDirectory()</slot>
  <slot>startAnalize()</slot>
 </slots>
</ui>
